CMakeLists.txt

```diff
@@ -441,6 +441,9 @@ if (OPM_ENABLE_PYTHON)
   # setup-install.sh - and completely bypass cmake in the installation phase.
   if (OPM_INSTALL_PYTHON)
     install( CODE "execute_process(COMMAND ${Python3_EXECUTABLE} ${PROJECT_BINARY_DIR}/python/install.py ${PROJECT_BINARY_DIR}/python/opm ${DEST_PREFIX}${CMAKE_INSTALL_PREFIX}/${PYTHON_INSTALL_PREFIX} 1)")
+    ## Need to install this Python script such that it can be used by opm-simulators when building against an installed
+    ##  opm-common
+    install( PROGRAMS "python/install.py" DESTINATION "share/opm/python" )
```

Comment:
You should probably install to "${CMAKE_INSTALL_DATAROOTDIR}/opm/python"  and export the location of the script in a variabke in opm-common-config.cmake as the user might overwrite the install path arbitrarily. Note that there are two versions of that file one that gets installed and the other in the build directory.